backend/src/db/payments.js

```diff
@@ -0,0 +1,26 @@
+const models = require('../models');
+const db = require('../sequalize_pg');
+
+const createPayment = async (paymentObj) => {
+    await db.sync();
+    models.payments.create(paymentObj);
+};
+
+const getPayment = async (ordernumber) => {
```

Comment:
Could be named to be `getPaymentByOrderNumber`. Also the order number is two words, so the arguments name should be `orderNumber`.